mrchecker-selenium-junit-concurrent-example/README.md

```diff
@@ -0,0 +1,51 @@
+MrChecker Selenium Junit concurrent
+-------------------
+
+This project is an example of usage MrChecker Selenium module with concurrent Junit.
+
+## How to use
+
+You can use this project to start your own:
+
+* Copy example project to your own location
+
+* Rename project (check also pom.xml values)
+
+* Replace example classes and test data
+
```

Comment:
Add information about the annotation that should be added to make tests parallel within one class